Caboodle/Browser/BrowserExtensions.shared.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public partial class Browser
```

Comment:
Let's get rid of this file for now. We can add one if we ever have extension methods to add to it.